src/main/common/components/free-mediation/freeMediationCheckAndSendBlock.njk

```diff
@@ -26,7 +33,7 @@
         bold = true
       ) }}
     {% endif %}
-    {% if mediationEnabled and mediationDraft.willYouTryMediation.option === YesNoOption.YES.option %}
+    {% if not enhancedMediationJourney and mediationEnabled and mediationDraft.willYouTryMediation.option === YesNoOption.YES.option %}
```

Comment:
why cant this go into another elseif or elif block ? so that we can rempve first two checks "if not enhancedMediationJourney and mediationEnabled"

Reply:
> why cant this go into another elseif or elif block ? so that we can rempve first two checks "if not enhancedMediationJourney and mediationEnabled"

Line no 36-42 will only appear if it is not an enhanced journey. These are 3 separate links in check-and-send page with different links.